examples/front-proxy/front-envoy.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
What is this leading `---` for?

Reply:
added by the converter, and apparently indicates "beginning of file". Probably harmless to remove them if it's irksome

Reply:
My mild preference is to remove them if you don't mind just because I think many people will be confused.